.github/workflows/test.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         with:
           python-version: '3.12.8'
 
+      - name: Install system dependencies (FFmpeg)
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ffmpeg
```

Comment:
I guess we should also add ffmpeg installion in the docker definition, shouldn't we?
And add a disclaimer in the readme that for people using conda they should run `conda install -c conda-forge ffmpeg` additionally (its a binary package so it cannot be added in the `requirements.txt` afaik, alternatively, we can add `imageio-ffmpeg` in the `requirements.txt` but I have not tested this and this might mess up with the docker definition)